modules/compliance-profiles.adoc

```diff
@@ -11,44 +12,39 @@ There are several profiles available as part of the Compliance Operator installa
 +
 [source,terminal]
 ----
-$ oc get -n <namespace> profiles.compliance
-----
-+
-This example displays the profiles in the default `openshift-compliance` namespace:
-+
-[source,terminal]
-----
 $ oc get -n openshift-compliance profiles.compliance
 ----
 +
 .Example output
 [source,terminal]
 ----
-NAME                 AGE
-ocp4-cis             32m
-ocp4-cis-node        32m
-ocp4-e8              32m
-ocp4-moderate        32m
-ocp4-moderate-node   32m
-ocp4-nerc-cip        32m
-ocp4-nerc-cip-node   32m
-ocp4-pci-dss         32m
-ocp4-pci-dss-node    32m
-rhcos4-e8            32m
-rhcos4-moderate      32m
-rhcos4-nerc-cip      32m
+NAME                             AGE
+ocp4-cis                         51m
+ocp4-cis-node                    51m
+ocp4-e8                          51m
+ocp4-high                        51m
+ocp4-high-node                   51m
+ocp4-moderate                    51m
+ocp4-moderate-node               51m
+ocp4-nerc-cip                    51m
+ocp4-nerc-cip-node               51m
+ocp4-pci-dss                     51m
+ocp4-pci-dss-node                51m
+ocp4-stig                        51m
+ocp4-stig-node                   51m
+rhcos4-anssi-bp28-enhanced       51m
+rhcos4-anssi-bp28-high           51m
+rhcos4-anssi-bp28-intermediary   51m
+rhcos4-anssi-bp28-minimal        51m
+rhcos4-e8                        51m
+rhcos4-high                      51m
+rhcos4-moderate                  51m
+rhcos4-nerc-cip                  51m
```

Comment:
The above profiles included a lot of profiles exist in upstream only. Should be: 

```
NAME                 AGE
ocp4-cis             94m
ocp4-cis-node        94m
ocp4-e8              94m
ocp4-high            94m
ocp4-high-node       94m
ocp4-moderate        94m
ocp4-moderate-node   94m
ocp4-nerc-cip        94m
ocp4-nerc-cip-node   94m
ocp4-pci-dss         94m
ocp4-pci-dss-node    94m
rhcos4-e8            94m
rhcos4-high          94m
rhcos4-moderate      94m
rhcos4-nerc-cip      94m
```